.github/workflows/publish-to-pypi.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     - uses: actions/checkout@v3
       with:
         fetch-depth: 0
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```

Comment:
Does this publishing action have anything to do with the mapreader code? If it doesn't perhaps you don't need to change

Reply:
I think this is the action that creates the build we publish to pypi [here](https://pypi.org/project/mapreader/#files).

I had assumed it would error if I left it on 3.8 but maybe not? Either way it might be safer to bump it to 3.9 since 3.8 is losing support in October.

Reply:
I'm not sure but sounds sensible!